.github/workflows/testimonial.yml

```diff
@@ -0,0 +1,50 @@
+name: Testimonial
+  
+on:
+  issues:
+    types: [opened]
+
+jobs:
+  create-testimonial:
+    runs-on: ubuntu-latest
+    name: Create Testimonial
+    steps:
+      - uses: renanmav/match-label-action@v3
+        with:
+          allowed: >
+            testimonial
+      - uses: actions/checkout@v3
+      - uses: stefanbuck/github-issue-parser@v3
+        id: issue-parser
+        with:
+          template-path: .github/ISSUE_TEMPLATE/testimonial.yml
+          body: ${{ github.event.issue.body }}
+      - run: cat ${HOME}/issue-parser-result.json
+      - name: Current Date
+        id: date
+        uses: Kaven-Universe/github-action-current-date-time@v1
+        with:
+          format: "YYYY-MM-DD"
+      - name: Create folder
+        run: |
+          mkdir -p data/${{ steps.issue-parser.outputs.issueparser_name }}
+      - uses: finnp/create-file-action@master
+        env:
+          FILE_NAME: "data/${{ steps.issue-parser.outputs.issueparser_name }}/${{ github.actor }}.json"
+          FILE_DATA: '{"title": "${{ steps.issue-parser.outputs.issueparser_title }}", "description": "${{ steps.issue-parser.outputs.issueparser_description}}", "date": "${{ steps.date.outputs.time }}"}'
+      - name: Setup Git config
+        run: |
+          # setup the username and email. I tend to use 'GitHub Actions Bot' with no email by default
+          git config user.name "GitHub Actions Bot"
+          git config user.email "<>"
+      - name: Commit
+        run: |
+          # setup the username and email. I tend to use 'GitHub Actions Bot' with no email by default
+          git add --all
+          git commit -m "New testimonial"
```

Comment:
I think it would be better here to include the `author` and the testimonial receiver 

For example...
> Testimonial for "eddiejaoude" by "Myrfion"

Reply:
Also this GitHub Action might help here https://github.com/marketplace/actions/git-auto-commit